polus-splinedist-inference-plugin/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.7 
```

Comment:
Since you want to use Tensorflow with a GPU, you are going to either want to use the Tensorflow base package, or the `bfio:2.1.9-tensorflow` base package. Otherwise your containerized plugin will not be able to use GPU (unless you go through the pain of installing all the necessary drivers inside the container on your own).